internal/service/project/resource_project_test.go

```diff
@@ -552,8 +552,7 @@ func TestAccProject_basic(t *testing.T) {
 	checks = acc.AddAttrSetChecks(dataSourceNameByID, checks, commonSetChecks...)
 	checks = acc.AddAttrSetChecks(dataSourceNameByName, checks, commonSetChecks...)
 	checks = append(checks, checkExists(resourceName), checkExists(dataSourceNameByID), checkExists(dataSourceNameByName))
-	// TODO: Uncomment this when CLOUDP-327589 is implemented
-	// checks = acc.AddAttrSetChecks(dataSourcePluralName, checks, "total_count", "results.#", "results.0.is_slow_operation_thresholding_enabled")
+	checks = acc.AddAttrSetChecks(dataSourcePluralName, checks, "total_count", "results.#", "results.0.is_slow_operation_thresholding_enabled", "results.0.users.#")
```

Comment:
have doubt if this check is actually ensuring there is at least one element. I know we are using `acc.IntGreatThan(0)` custom function in other places.